webonary-cloud-api/lambda/browseEntries.ts

```diff
@@ -20,37 +34,84 @@ export async function handler(
 
     const dictionaryId = event.pathParameters?.dictionaryId;
     const text = event.queryStringParameters?.text;
-    const lang = event.queryStringParameters?.lang;
+    const mainLang = event.queryStringParameters?.mainLang; // main language of the dictionary
+    const lang = event.queryStringParameters?.lang; // this is used to limit which language to search
+
+    const countTotalOnly = event.queryStringParameters?.countTotalOnly;
+
+    const pageNumber = Math.max(Number(event.queryStringParameters?.pageNumber ?? ''), 1);
```

Comment:
So refresh my memory on what the Number global function does with ''? I think '0' would be clearer to me.

Reply:
defaulted to '1'